src/Symfony/Component/Form/Tests/ChoiceList/Factory/PropertyAccessDecoratorTest.php

```diff
@@ -63,6 +63,21 @@ public function testCreateFromChoicesPropertyPathInstance()
         $this->assertSame(array('value'), $this->factory->createListFromChoices($choices, new PropertyPath('property')));
     }
 
+    /**
+     * @group legacy
+     */
+    public function testCreateFromChoicesPropertyPathWithCallableString()
+    {
+        $choices = array(array('end' => 'value'));
+
+        $this->decoratedFactory->expects($this->once())
+            ->method('createListFromChoices')
+            ->with($choices, 'end')
+            ->will($this->returnValue(array('value')));
```

Comment:
I think this test case is a bit confusing. In fact, the actal values of the choices and return values don't matter here. This test suggests they do, however.

I'd set `$choices` to some arbitrary value, to make that clear, e.g. `array('foo', 'bar')`.

I'd also set the return value of this line to `'RESULT'` to make it clear that the result of the underlying implementation is simply passed through.